src/components/SearchField.tsx

```diff
@@ -87,28 +91,44 @@ const SearchField: React.FC = () => {
   const fromDateInputIncrease = (
     event: React.MouseEvent<HTMLButtonElement, MouseEvent>
   ): void => {
-    if (dateRange.fromYear >= dateRange.toYear) return;
+    if (dateRange.fromYear >= dateRange.toYear) {
+      setIsYearDecrease(true);
+      return;
+    }
+    setIsYearDecrease(false);
     setDateRange({ ...dateRange, fromYear: dateRange.fromYear + 1 });
   };
 
   const fromDateInputDecrease = (
     event: React.MouseEvent<HTMLButtonElement, MouseEvent>
   ): void => {
-    if (dateRange.fromYear <= lowerDateLimit) return;
+    if (dateRange.fromYear <= lowerDateLimit) {
+      setIsYearDecrease(true);
+      return;
+    }
+    setIsYearDecrease(false);
     setDateRange({ ...dateRange, fromYear: dateRange.fromYear - 1 });
   };
 
   const toYearInputIncrease = (
```

Comment:
Can you please rename functions so it named in same way? toDateInputDecrease, toDateInputIncrease

Reply:
Got it. Already change